includes/promoted-jobs/class-wp-job-manager-promoted-jobs-api.php

```diff
@@ -234,6 +233,7 @@ private function prepare_item_for_response( WP_Post $item ) {
 		return [
 			'id'           => (string) $item->ID,
 			'status'       => $item->post_status,
+			'promoted'     => WP_Job_Manager_Promoted_Jobs::is_promoted( $item->ID ),
```

Comment:
Currently, I don't think so. But I added it because it can be useful in the future and make the feed more concise (we don't have promoted and not promoted jobs in the feed without a differentiation).
We could use that, in case we want to list all the promoted jobs as a type of extra promotion, for example.

Reply:
Sounds good, thanks!